react/components/Page.jsx

```diff
@@ -1,22 +1,23 @@
-var React = require('react');
-var Link = require('react-router').Link;
-var Sidebar = require('./Sidebar.jsx');
+var React = require(`react`);
+var Link = require(`react-router`).Link;
+var Sidebar = require(`./Sidebar.jsx`);
 
 var elements = (function() {
-  var randomstring = require('randomstring');
-  var elements = [];
+  var randomstring = require(`randomstring`);
+  var elems = [];
   var i=100;
+
   while(i--) {
-    elements.push(randomstring.generate());
+    elems.push(randomstring.generate());
   }
-  return elements;
-}());
+  return elems;
```

Comment:
That makes sense as an error to me. You had `elements` declared in global scope and then were redeclaring it inside a closure. Generally that's inadvisable as it can lead to confusion as to which instance of that variable name some given code is operating on.